client/app/components/onboardingGuide/GuideCoverPhotoPage.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import scss from './styles.scss';
+
+const GuideCoverPhotoPage = (props) => {
+
+  const handleClick = function(e, path) {
+    e.preventDefault();
+    props.changePage(path);
+  }
+
+  return (
+    <div className="container">
+      <a className={ scss.backLink }
+        onClick={ (e) => handleClick(e, "") }
+        href={ props.initialPath } >
+        ‹ { props.t('back_to_todo') }
+      </a>
+
+      <h2 className={ scss.title } >{ props.t('title') }</h2>
+
+      <p className={ scss.description } >
+        { props.t('description') }
+      </p>
+
+      <div className={ scss.sloganImageContainer } >
+        <img src={ props.pageData.infoImage } className={ scss.sloganImage } />
+      </div>
+
+      <div className={ scss.infoTextContainer } >
+        <div className={ scss.infoTextIcon } >
+          <i className="ss-info"></i>
+        </div>
+        <div className={ scss.infoTextContent } >
+          { props.t('advice_p1') }
+          <br />
```

Comment:
Maybe two `<p>` tags instead of a single `<div>` with a `<br/>` in it.

PS: and I know old habits die hard, but there is really no need to have the space before the self-closing tag `/` anymore ;)